docs/requirements/stakeholder/index.rst

```diff
@@ -17,6 +17,739 @@
 Stakeholder Requirements
 ========================
 
+Overall goals
+-------------
+
+.. stkh_req:: Reuse of application software via managed APIs
+   :id: STKH_REQ__20
+   :reqtype: Non-Functional
+   :security: NO
+   :safety: QM
+   :rationale: This is a usability constraint needed for long maintenance support
+   :status: valid
+
+   The platform shall enable the reuse of application software via a set of managed APIs.
+   These APIs shall be developed via a well-defined life-cycle ensuring
+   non-breaking changes.
+
+
+.. stkh_req:: Enable cooperation via standardized APIs
+    :id: STKH_REQ__30
+    :reqtype: Non-Functional
+    :security: NO
+    :safety: QM
+    :rationale: To enable cooperation with other cooperation partners.
+    :status: valid
+
+    The software platform shall be based on open and/or standardized APIs.
+
+.. stkh_req:: Variant management
+    :id: STKH_REQ__60
+    :reqtype: Functional
+    :security: NO
+    :safety: QM
+    :rationale: tbd
+    :status: valid
+
+    The software platform shall provide variant management support.
+    Variant management support shall enable users to ensure the
+    compatibility of application software across vehicle variants and vehicle
+    software releases.
+
+
+.. stkh_req:: IP protection
+   :id: STKH_REQ__50
+   :reqtype: Non-Functional
+   :security: NO
+   :safety: QM
+   :rationale: tbd
+   :status: valid
+
+   The software platform shall support cooperation models, where partners do not
```

Comment:
same here what does that mean?

Reply:
There might be projects where the sw platform is hosting closed source applications.